static/coffee/services.coffee

```diff
@@ -251,12 +251,14 @@ services.factory('CaseService', ['$http', '$httpParamSerializer', '$window', ($h
     #----------------------------------------------------------------------------
     # Opens a new case
     #----------------------------------------------------------------------------
-    open: (message, summary, assignee, user) ->
-      params = {message: message.id, summary: summary}
+    open: (message, summary, assignee, user, urn) ->
+      params = {summary: summary, urn: urn}
```

Comment:
Is urn going to be given when cases are created with an initial message as well? Or is it still optional even if though it's in this object?